### PR TITLE
Mod1

### PR DESCRIPTION
**Pull request by kcny**
Final pull request for week 2.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
   resources :videos, except: [:destroy] do 
     collection do
       post :search, to: "videos#search"
     end
-  resources :reviews, only: [:create]  
-  end
-  get 'ui(/:action)', controller: 'ui'
+  resources :reviews, only: [:create]
+end
+  #resources :videos, only:  [:show]
```
Comment (alagram) [inline config/routes.rb]: You might want to remove this comment if its not in use.

File: config/routes.rb [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+
+  get 'ui(/:action)', controller: 'ui'
+  get 'register',       to: 'users#new'
+  get '/login',       to: 'sessions#new'
+  post '/login',        to: 'sessions#create'
+  get '/logout',        to: 'sessions#destroy'
+  
   resources :users, only: [:create] 
   resources :sessions, ony: [:create]
+  #resources :queue_items, only: [:index]
```
Comment (alagram) [inline config/routes.rb]: Same here.

File: config/routes.rb (hunk 1 above)
Comment (kcny) [inline config/routes.rb]: Done, thanks for catching that.

File: config/routes.rb (hunk 2 above)
Comment (kcny) [inline config/routes.rb]: Done, thanks!

**Pull request merged**